README.md

```diff
@@ -122,51 +88,27 @@ DOCKER_API_VERSION=1.23
 export DOCKER_API_VERSION
 ```
 
-For convenience, a script called "run.sh" is provided which can be edited to specify the desired options.
-
+## Usage
 
-Run Container
-------------
-The best way to experiment with the driver is to run it as a container.
-
-A pre-built docker image can be pulled from Docker Hub using the following command:
-```
-docker pull infoblox/ipam-driver
-```
-
-After successfully pulling the image, you use the ```docker run``` command to run the driver. For exampe:
-```
-docker run -e DOCKER_API_VERSION=1.22 -v /var/run:/var/run -v /run/docker:/run/docker infoblox/ipam-driver --grid-host=192.168.124.200 --wapi-username=cloudadmin --wapi-password=cloudadmin --local-view=local_view --local-network-container="192.168.0.0/20,192.169.0.0/22" --local-prefix-length=25 --global-view=global_view --global-network-container="172.18.0.0/16" --global-prefix-length=24
-```
-
-Note that the -v options are necessary to provide the container access to the specified directories on the
-host file system.
-
-For convenience, a script called "run-container.sh" is provided.
-
-Usage
------
 To start using the driver, a docker network needs to be created specifying the driver using the --ipam-driver option:
 ```
-sudo docker network create --ipam-driver=infoblox priv-net
+$ docker network create --ipam-driver=infoblox/ipam-plugin:v1.1.0 priv-net
```

Comment:
I don't think so we can create multiple instances of the plugin. I tried installing a plugin twice and it gives error.
```
$ docker plugin ls
ID                  NAME                        DESCRIPTION                       ENABLED
d9d00a5239f2        ishant8/ipam-plugin:1.1.0   Infoblox IPAM plugin for Docker   true

$ docker plugin install ishant8/ipam-plugin:1.1.0
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
Error response from daemon: plugin "ishant8/ipam-plugin:1.1.0" already exists
```

There is a option **--alias** while installing the plugin and it creates the plugin with different name. 
```
$ docker plugin install ishant8/ipam-plugin:1.1.0 --alias infoblox
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
1.1.0: Pulling from ishant8/ipam-plugin
32f625173489: Download complete
Digest: sha256:525ded62dd7bf6920b796d32f931808c9403a0a1ee76d07c8657d3bfacc3a1aa
Status: Downloaded newer image for ishant8/ipam-plugin:1.1.0
Installed plugin ishant8/ipam-plugin:1.1.0

$ docker plugin install ishant8/ipam-plugin:1.1.0 --alias infoblox-2
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
1.1.0: Pulling from ishant8/ipam-plugin
32f625173489: Download complete
Digest: sha256:525ded62dd7bf6920b796d32f931808c9403a0a1ee76d07c8657d3bfacc3a1aa
Status: Downloaded newer image for ishant8/ipam-plugin:1.1.0
Installed plugin ishant8/ipam-plugin:1.1.0

$ docker plugin ls
ID                  NAME                DESCRIPTION                       ENABLED
0170bd86a366        infoblox:latest     Infoblox IPAM plugin for Docker   true
5057272f83d7        infoblox-2:latest   Infoblox IPAM plugin for Docker   true
```

**But the use case of --alias is different**. It is used for the seamless upgrades and get rid of the version dependency. For eg if we install plugin using full name - `docker plugin install ishant8/ipam-plugin:1.1.0` and then create a network with this ipam-plugin - `docker network create --ipam-plugin ishant8/ipam-plugin:1.1.0 net-1` , then net-1 is bound to this version of plugin. Upgrading with the `docker plugin upgrade ..` will cause issues as this network needs to be deleted first. 

So I dont think we can support multiple instances of ipam plugin on same host.